deploy/roles/eod/defaults/main.yaml

```diff
@@ -29,3 +33,6 @@ gconf_settings:
     setting: "/apps/gdm/simple-greeter/disable_user_list"
     type: "bool"
     value: "true"
+
+eod_DisplaysPerHost:        100
+eod_Xconfig_directory:      /usr/local/eod/conf/admin/Xconfig
```

Comment:
@olivermel Are all of these formatting changes necessary?  Just want to make sure they won't affect the actual output in templates, etc.